<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>652</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonOpenImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_Thr1">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>10</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewImage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>521</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Thr1">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Thr2">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>40</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_apertureSize">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>70</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar_Thr2">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>40</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSaveCannyImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Canny Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Second threshold for the hysteresis procedure</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewCannyImage">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>521</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalScrollBar_Thr1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Thr1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Thr1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalScrollBar_Thr1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_Thr2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Thr2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_Thr2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalScrollBar_Thr2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
